app/lib/import_export_constants.rb

```diff
@@ -134,6 +135,7 @@ module ImportExportConstants # rubocop:todo Metrics/ModuleLength
     times: %i[preferred_contact_time],
     timestamps: %i[created_at updated_at closed_at latest_assessment_at latest_transfer_at last_assessment_reminder_sent],
     phones: %i[primary_telephone secondary_telephone],
+    int_phones: %i[international_telephone],
```

Comment:
Does this need to be its own field type? Or can it just be a string?

Reply:
the reason it's not a `phone` type is because we do not apply the same validations we do for domestic phone numbers, the reason it's not a `string` type is because we still do some validation still (only allowing certain characters and digits)

Reply:
I'm not seeing where this object is used in the validations though, that's why I was thinking it could just be string? Since this object appears to be for how something is exported.

Reply:
you are right, removed: https://github.com/SaraAlert/SaraAlert/pull/1069/commits/88ac0435a48b054947d19cbf9c2da2993c286ac9#diff-4f61e388997f8d92e41f66947a47b461707b615cf9fac69f1080b8254630d2ceL258